GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -807,7 +807,7 @@ private bool DeleteSelectedFiles()
                     {
                         Directory.Delete(path, true);
                     }
-                    else
+                    else if (!isDir)
```

Comment:
This condition will always be fullfilled, doesn't it?

Reply:
Sure